app/assets/stylesheets/signup_customer.scss

```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css?family=Calibri');
+
+body, html {
+  font-family: 'Calibri', serif;
+  }
+
+.main-section{
+  background-color: #45C4C9;
+  background-size:100% 100%;
+  margin: 0px;
+  width: 100%;
+}
+
+.user-login-header,.login-image-main{
+    margin-top: 15px;
+}
+.user-login-header h1{
+  font-size: 60px;
+  color:#EFF8E2;
+}
+.user-login-header p,.last-part p{
+  color:#087E8B;
+  font-size: 20px
+}
+.user-login-header span{
+  font-weight: 600;
+  font-size: 30px;
+}
+.login-image-main{
+  padding: 30px;
+  background-color:#FCF1EE;
+  border-radius:5px;
+  box-shadow: 0 0 5px 0 #fff;
+  margin-bottom:20px;
+}
+.user-image-section img{
+  background-color: #087E8B;
+  padding:30px;
+  height:150px;
+  width:150px;
+  border-radius: 50%;
+}
+.user-login-box,.last-part{
+  padding:20px;
+}
+.user-login-box a,.user-login-box a:hover{
+  background-color: #087E8B;
+  border:1px solid #45C4C9;
+  width: 100%;
+  color:#fff;
+  font-weight:600;
+}
+.last-part a{
+  text-decoration: none;
+  color:#087E8B;
+}
+
+.last-part a span{
+font-weight:bold;
+}
+
+.last-part a span:hover{
+color:#45C4C9
+}
+
+.btn{
+  color: #EFF8E2;
+  background-color: #087E8B;
+  border-color: #087E8B;
```

Comment:
korzystając z scss można tworzyć zmienne. Możesz dodać np partial _colors.scss, zaincludować go w Twoim głównym arkuszu stylów i wrzucić tam kolory i pododawać im nazwy. Wtedy wszędzie gdzie używasz jednego koloru używasz tej zmiennej i jak będziesz chciała zmienić ten kolor na inny to podmieniasz go tylko w jednym miejscu

Reply:
zgadzam się, że zmiennymi można by tu wiele zdziałać, bo większość właściwości powtarza mi się w wielu widokach. postaram się to zastosować